src/ints/bios.cpp

```diff
@@ -1021,23 +1021,28 @@ static Bitu INT15_Handler(void) {
 
 static Bitu Default_IRQ_Handler(void) {
 	IO_WriteB(0x20,0x0b);
-	Bit8u master_isr=IO_ReadB(0x20);
-	if (master_isr) {
-		IO_WriteB(0xa0,0x0b);
-		Bit8u slave_isr=IO_ReadB(0xa0);
-		if (slave_isr) {
-			IO_WriteB(0xa1,IO_ReadB(0xa1)|slave_isr);
-			IO_WriteB(0xa0,0x20);
-		} else IO_WriteB(0x21,IO_ReadB(0x21)|(master_isr&~4));
-		IO_WriteB(0x20,0x20);
+	uint8_t primary_isr = IO_ReadB(0x20);
```

Comment:
What is ISR? Are we sure we should name it as primary/secondary?

Reply:
These are PICs, but implemented in BIOS software: Interrupt Service Routine:

![2021-07-08_18-23](https://user-images.githubusercontent.com/1557255/125009612-b62ae200-e019-11eb-9187-44262ca75d58.png)

Even more - there isn't even a reference to master or slave, so this terminology was likely used to mirror the PIC controllers.

I'll add a small note near the top. (Fixup inbound).